cvat-sdk/cvat_sdk/core/downloading.py

```diff
@@ -58,6 +61,17 @@ def download_file(
             except ValueError:
                 file_size = None
 
+            if output_path.is_dir():
+                content_disposition = next(
+                    part.strip()
+                    for part in response.headers.get("Content-Disposition", "").split(";")
+                    if part.strip().startswith("filename=")
+                )
```

Comment:
What if there is no filename, or no `Content-Disposition` header at all? I think we should produce an exception with an understandable message in this case.

Reply:
It this case the tests should fail, but I added error handling.